# Lesson 5

### Задание 1
#### Известно, что генеральная совокупность распределена нормально со средним квадратическим отклонением, равным 16.
##### Найти доверительный интервал для оценки математического ожидания $\, a \,$ с надежностью 0.95, если выборочная средняя M = 80, а объем выборки n = 256.

#### Решение:

$$T_{1,2} = \overline{X} \pm \frac{s_0}{\sqrt{n}} \cdot c_{\gamma}$$

$$T_{1,2} = 80 \pm \frac{16}{\sqrt{256}} \cdot 1.96$$

$$T_{1,2} = 80 \pm 1.96 = 79.04,\, 81.96$$

#### Ответ: 79.04 и 81.96

### Задание 2
#### В результате 10 независимых измерений некоторой величины X, выполненных с одинаковой точностью, получены опытные данные: 
6.9, 6.1, 6.2, 6.8, 7.5, 6.3, 6.4, 6.9, 6.7, 6.1
##### Предполагая, что результаты измерений подчинены нормальному закону распределения вероятностей, оценить истинное значение величины X при помощи доверительного интервала, покрывающего это значение с доверительной вероятностью 0,95.

#### Решение:

In [1]:
import math
import numpy as np
import pandas as pd

In [2]:
df1 = pd.DataFrame([6.9, 6.1, 6.2, 6.8, 7.5, 6.3, 6.4, 6.9, 6.7, 6.1], columns=['my_data'])

In [3]:
df1_mean = round(df1['my_data'].mean(), 2)
df1_mean

6.59

In [4]:
df1_var = round(df1['my_data'].var(ddof=1), 4)
df1_var

0.2032

In [5]:
df1_sigma = round(math.sqrt(df1_var), 4)
df1_sigma

0.4508

In [6]:
result2_1 = df1_mean-round((df1_sigma/math.sqrt(df1['my_data'].count()))*2.262, 4)
result2_2 = df1_mean+round((df1_sigma/math.sqrt(df1['my_data'].count()))*2.262, 4)
result2_1, result2_2

(6.2675, 6.9125)

#### Ответ: 6.2675 и 6.9125

### Задание 3
#### Утверждается, что шарики для подшипников, изготовленные автоматическим станком, имеют средний диаметр 17 мм.
##### Используя односторонний критерий с $\alpha$=0,05, проверить эту гипотезу, если в выборке из n=100 шариков средний диаметр оказался равным 17.5 мм, а дисперсия известна и равна 4 кв. мм.

#### Решение:

In [7]:
Z_check = (17.5-17)/(math.sqrt(4)/math.sqrt(100))
Z_check

2.5

#### Ответ: $Z$ расчетная (2.5) справа от $Z$ табличной (1.65), поэтому верно среднее значение из выборки (17.5 мм - $\mu_1$) 

### Задание 4
#### Продавец утверждает, что средний вес пачки печенья составляет 200 г. Из партии извлечена выборка из 10 пачек. Вес каждой пачки составляет:
202, 203, 199, 197, 195, 201, 200, 204, 194, 190.
#### Известно, что их веса распределены нормально.
##### Верно ли утверждение продавца, если учитывать, что доверительная вероятность равна 99%?

#### Решение:

In [8]:
df2 = pd.DataFrame([202, 203, 199, 197, 195, 201, 200, 204, 194, 190], columns=['my_weight'])

In [9]:
df2_mean = round(df2['my_weight'].mean(), 2)
df2_mean

198.5

In [10]:
df2_sigma = round(math.sqrt(df2['my_weight'].var(ddof=1)), 4)
df2_sigma

4.4535

In [11]:
t_check = round((df2_mean-200)/(df2_sigma/math.sqrt(10)), 4)
t_check

-1.0651

#### Ответ:  $t$  расчетная (-1.0651) слева от  $t$  табличной (2.821), поэтому верно среднее значение веса 200 г. ($\mu_0$)